static/index.html

```diff
@@ -423,6 +423,26 @@ <h3 class="header mb-2">Add panel to dashboard</h3>
                 <button type="button" id="create-panel" class="btn create-db">Create Panel</button>
             </div>
         </div>
+        <div class="addrulepopupOverlay">
+            <div class="addrulepopupContent">
+                <h3 class="header" style="font-size: 18px;text-align: left;font-family: 'DINpro';font-weight: 600;">Create Alert</h3>
```

Comment:
1. Avoid defining styles directly in HTML.
2. Update the CSS for dark mode.
<img width="1426" alt="image" src="https://github.com/siglens/siglens/assets/71771131/915daa75-616b-4525-8cd3-e320ae031d5a">

3. Prevent creating alerts with empty rule names.
4. Remove any transformations or delays when popups open.
5. Adjust the height of the header and buttons to match our UI. Refer to other popups for reference.